src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -161,9 +161,7 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
 
   const allowAdvertising = pageData?.metadata?.allowAdvertising ?? false;
   const adcampaign = pageData?.metadata?.adCampaignKeyword;
-  const isTransliterated =
-    ['serbian', 'zhongwen', 'uzbek'].includes(service) &&
-    pageType === ARTICLE_PAGE;
+  const isUzbekArticle = service === 'uzbek' && pageType === ARTICLE_PAGE;
```

Comment:
Are we just ignoring Uzbek for now - and focussing on Zhongwen + Serbian?

Reply:
Yea Uzbek will follow later once the CMS supports transliteration for Topics.